dumpling/tests/e2e/run.sh

```
@@ -37,24 +37,85 @@ run_lightning $cur/conf/lightning.toml
+    run_sql "drop database if exists $DB_NAME;"
+
+    export DUMPLING_TEST_PORT=3306
+
+    # dumping
+    export DUMPLING_TEST_DATABASE=$DB_NAME
+    rm -rf "$DUMPLING_OUTPUT_DIR"
+    run_dumpling --compress "$compress_type"
+
+    cat "$cur/conf/lightning.toml"
+    # use lightning import data to tidb
+    run_lightning $cur/conf/lightning.toml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)